ccd-definition/AuthorisationCaseField/gatekeeper.json

```diff
@@ -270,21 +270,21 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "amountToPay",
     "UserRole": "caseworker-publiclaw-gatekeeper",
-    "CRUD": "CRU"
+    "CRUD": "R"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "displayAmountToPay",
     "UserRole": "caseworker-publiclaw-gatekeeper",
-    "CRUD": "CRUD"
+    "CRUD": "R"
   },
   {
```

Comment:
Do gatekeepers need permissions on these fields, they are used in the submission and upload c2 events and gatekeeper only has read permissions for that event

Reply:
agreed, can remove completely